migrations/migration_constants.js

```diff
@@ -33,7 +33,9 @@ const NAMES_OF_CONTRACTS_OWNED_BY_MULTISIG = [
     "TokenTransferProxy",
     "RepaymentRouter",
     "Collateralizer",
+    "ERC721Collateralizer",
     "TokenRegistry",
+    "ERC721TokenRegistry",
```

Comment:
As discussed on Slack, I think the transfer of these two new contracts to the multisig should happen at a later migration so as to give us room (potentially also in another migration) to set all of the token attributes for the initial batch.

Reply:
*without invoking the 7-day timelock